---
title: "Research and Publications"
---

***

### Current Research

My research interests include statistical computing environments and methods for big, complex data, especially datasets which have non-trivial correlation structures or which integrate data from multiple sources. These types of datasets are common in many fields, particularly biomedical research and natural language processing. They pose a challenge to traditional computing and statistical approaches not only because of their size, but also because of their complexity.

My current research work is in machine learning and statistical computing for bioinformatics. I develop statistical computing infrastructure for analysis of complex, high-dimensional experiments in genomics, proteomics, lipidomics, and metabolomics. Specifically, I develop statistical computing frameworks for high-dimensional biological imaging experiments.

My current work focuses on applications of mass spectrometry (MS) imaging experiments, which has inspired the development of my R packages [_Cardinal_](https://cardinalmsi.org) and _matter_. I am particularly interested in developing methods of statistical analysis for experiments that integrate both MS imaging and other types of biomedical imaging modalities, such as spatial transcriptomics and microscopy.

***

### Recent Publications

- S. S. Lakkimsetty, A. Weber, **_K. A. Bemis_**, V. Stehl, P. Bronsert, M. C. Foell, and O. Vitek. "MSIreg: an R package for unsupervised coregistration of mass spectrometry and H&E images."  _Bioinformatics_. 2024. https://doi.org/10.1093/bioinformatics/btae624

- **_K. A. Bemis_**, M. C. Foell, D. Guo, S. S. Lakkimsetty, and O. Vitek. "Cardinal v3 - a versatile open source software for mass spectrometry imaging analysis."  _Nature Methods_. 2023. https://doi.org/10.1038/s41592-023-02070-z

- D. Guo, M. C. Foell, **_K. A. Bemis_**, and O. Vitek. "A noise-robust deep clustering of biomolecular ions improves interpretability of mass spectrometric images."  _Bioinformatics_. 2023. https://doi.org/10.1093/bioinformatics/btad067

- D. Guo, **_K. A. Bemis_**, C. Rawlins, J. Agar, and O. Vitek. "Unsupervised segmentation of mass spectrometric ion images characterizes morphology of tissues."  _Bioinformatics_. 2019. http://doi.org/10.1093/bioinformatics/btz345

- **_K. A. Bemis_**, D. Guo, A. Harry, M. Thomas, I. Lanekoff, M. Stenzel-Poore, S. Stevens, J. Laskin, and O. Vitek. "Statistical detection of differentially abundant ions in mass spectrometry-based imaging experiments with complex designs." _International Journal of Mass Spectrometry_. 2019. http://doi.org/10.1016/j.ijms.2018.07.006

- **_K. A. Bemis_** and O. Vitek. (2017). "_matter_: an R package for rapid prototyping with larger-than-memory datasets on disk." _Bionformatics_. 2017. http://doi.org/10.1093/bioinformatics/btx392

- **_K. D. Bemis_**, A. Harry, L. S. Eberlin, C. Ferreira, S. M. van de Ven, P. Mallick, M. Stolowitz, and O. Vitek. (2016). "Probabilistic Segmentation of Mass Spectrometry (MS) Images Helps Select Important Ions and Characterize Confidence in the Resulting Segments." _Molecular & Cellular Proteomics_. 2016. http://doi.org/10.1074/mcp.O115.053918

- S. van de Ven, **_K. D. Bemis_**, K. Lau, R. Adusumilli, U. Kota, M. Stolowitz, O. Vitek, P. Mallick, S. S. Gambhir. (2016). "Protein biomarkers on tissue as imaged via MALDI mass spectrometry: A systematic approach to study the limits of detection." _Proteomics_. 2016. http://doi.org/10.1002/pmic.201500515

- **_K. D. Bemis_**, A. Harry, L. S. Eberlin, C. Ferreira, S. M. van de Ven, P. Mallick, M. Stolowitz, and O. Vitek. "_Cardinal_: an R package for statistical analysis of mass spectrometry-based imaging experiments." _Bioinformatics_. 2015. http://doi.org/10.1093/bioinformatics/btv146

***

### Future Interests

My long-term research goals are informed both by my interest in statistical computing for big, complex data and by my identity as an indigenous trans woman. I am interested in leveraging computational linguistics and natural language processing techniques in the research and preservation of Native American languages, and in creating Zuni language learning resources for my tribe. This will entail development of a digital dictionary, digitization of existing texts, conversion to a common orthography, and tagging as a dual-lingual corpus for use with machine learning techniques. It will be necessary to integrate written and spoken language information, incorporate indigenous ways of knowing, and develop new digital ethics practices for culturally sensitive corpora.

I am also interested in advancing biomedical research for women and the LGBTQ community by developing new statistical methods for data integration. Although we know hormone replacement therapy affects gene expression and protein expression, there has been little comprehensive investigation into its effects on the proteome and epigenome, especially as it affects both transgender and cisgender women. The effect of variations in regimen, such as inclusion of progesterone, have not been thoroughly scientifically investigated, and imaging technologies and consideration of mental well-being have rarely been incorporated. I am interested in designing medical studies and developing analytic methods that can provide insight into creating better hormone therapy regimens, better understanding of hormone cycling in both trans and cis women, and enabling lactation for non-gestational parents with breasts.
